### PR TITLE
Move common includes into vivian_common.php

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
* Rename 'vivian_common_header.php' to 'vivian_common.php'
* Rename 'vivian_osehra_image.php' to 'vivian_header.php'

https://issues.osehra.org/browse/VIVIAN-530

### CODE REVIEW

File: Visual/bff_demo.php [hunk 1]
```diff
@@ -3,8 +3,9 @@
   <title>VHA Business Function Framework and Requirements</title>
   <head>
     <?php
-      include_once "vivian_common_header.php";
       include_once "vivian_tree_layout.css";
+      include_once "vivian_tree_layout_common.js"
```
Comment (josephsnyder) [inline Visual/bff_demo.php]: This is not the place to "include_once" of Javascript code.  If it isn't found within a <script> tag, it'll assume that it's HTML and try to display it on the page

Comment (BetsyMcPhail) [reply Visual/bff_demo.php]: I think these should all be moved now

**Comment by BetsyMcPhail** [comment]
PTAL when you have a chance @josephsnyder

**Comment by josephsnyder** [comment]
Everything looks good!

**Pull request merged**